cinema/urls.py

```diff
@@ -1 +1,19 @@
-# write urls here
+from django.urls import path, include
+from rest_framework import routers
+
+from cinema import views
+
+
+router = routers.DefaultRouter()
+
+router.register("cinema_halls", views.CinemaHallViewSet)
+router.register("genres", views.GenreViewSet)
+router.register("actors", views.ActorViewSet)
+router.register("movies", views.MovieViewSet)
+router.register("movie_sessions", views.MovieSessionViewSet)
+
+urlpatterns = [
+    path("", include(router.urls))
+]
```

Comment:
When basepath is empty no need to include, instead use:
urlpatterns = router.urls